pom.xml

```diff
@@ -169,13 +169,12 @@
                     <imageManagement>
                         <image>
                             <repository>${dockerHubPublic}/cafapi/opensuse-jre17</repository>
-                            <tag>4.0.0</tag>
-                            <digest>sha256:c0065ce7db226f864099ffd52371ebcf580c944a772f24fc0cb69ccacd9e6059</digest>
+                            <tag>4.0.1</tag>
+                            <digest>sha256:434c16f975fbbe2f4adfc879d88a32b5f8fcc92a2d8e29e3598dc10088717405</digest>
                         </image>
                         <image>
                             <repository>${dockerHubPublic}/opensearchproject/opensearch</repository>
                             <tag>2.14.0</tag>
-                            <digest>sha256:96af4ace999e20f3f74b1675e501d7dba46f2e7c185cfcffd4626898b00e6743</digest>
```

Comment:
@dermot-hardy I don't think the opensearch images are static, when I pull locally I get the sha above but when I look in dockerhub the sha is different and the docker plugin also finds a different sha when running use-latest-releases.

Reply:
Not part of this ticket.